pkg/scheduler/api/devices/nvidia/vgpu/utils.go

```diff
@@ -180,11 +180,7 @@ func decodePodDevices(str string) []ContainerDevices {
 
 func checkVGPUResourcesInPod(pod *v1.Pod) bool {
 	for _, container := range pod.Spec.Containers {
-		_, ok := container.Resources.Limits[VolcanoVGPUMemory]
-		if ok {
-			return true
-		}
-		_, ok = container.Resources.Limits[VolcanoVGPUNumber]
+		_, ok := container.Resources.Limits[VolcanoVGPUNumber]
```

Comment:
What if user just set VolcanoVGPUNumber but not set VolcanoVGPUMemory？

Reply:
it will try to find an unused GPU, and use that exclusively, if such can't be find, then it will be stuck in pending state.